app/routes/__public/oauth/$id.tsx

```diff
@@ -245,21 +243,16 @@ const getAnonymousAccessToken = async ({
     }))
     .catch((e) => ({ body: e.message, success: false as const }));
 
-  console.log("response", response);
-
   if (response.success) {
     try {
-      await cxn
-        .insert(accessTokens)
-        .values({
-          uuid: sql`UUID()`,
-          value: response.body.accessToken,
-          code,
-          installationId: installation_id || "",
-          userId: installation_id || "",
-          state: state || "",
-        })
-        .onDuplicateKeyUpdate({ set: { value: response.body.accessToken } });
```

Comment:
I was hoping to keep this - let's raise it tm

Reply:
Discussed.  As this is just for `anon=true` we can remove it.